packages/app/tailwind.config.js

```diff
@@ -1,17 +1,21 @@
-
 module.exports = {
   future: {},
   purge: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
   theme: {
     gradientColorStops: theme => ({
+      ...theme('colors'),
```

Comment:
Just for my own knowledge: Where does tailwind get these theme colors from?

Reply:
with the ...theme 'colors' you unpack all the colors from the tailwind.base.css class and expand it with your own options

Reply:
thx